eslzArm/managementGroupTemplates/policyAssignments/ENFORCE-ACSBPolicyAssignment.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
🐪 the file name please

Reply:
I have updated the filename but seems git doesn't care :)